ArduCopter/Copter.cpp

```diff
@@ -504,8 +508,12 @@ void Copter::ten_hz_logging_loop()
     if (should_log(MASK_LOG_ATTITUDE_MED) && !should_log(MASK_LOG_ATTITUDE_FAST) && !copter.flightmode->logs_attitude()) {
         Log_Write_Attitude();
     }
+    if (!should_log(MASK_LOG_ATTITUDE_FAST)) {
+    // log at 10Hz if PIDS bitmask is selected, even if no ATT bitmask is selected; logs at looprate if ATT_FAST and PIDS bitmask set
```

Comment:
I really don't think there is any point in logging PIDs without attitude at 10hz. What is the use case?

Reply:
just the overall logic of using a bitmask to allow what is being logged....nothing prevents someone from not wanting to log attitude and just logging pids to see the tar/act...if I check or uncheck a box, I would expect it to operate as labelled....right now if you check PID you only get them if an ATTITUDE box is checked....to me that is not how any other log option check box works...guess we could change the label to "PID logs if ATTITUDE logging"